shopinvader/models/shopinvader_variant.py

```diff
@@ -37,7 +37,7 @@ class ShopinvaderVariant(models.Model):
         check_company=True,
     )
     record_id = fields.Many2one(
-        string="Product",
+        string="Product Name",
```

Comment:
I'd prefer "Related product" but it's my POV :)

Reply:
good point ;)